modules/core/src/lib/composite-layer.js

```diff
@@ -38,6 +38,12 @@ export default class CompositeLayer extends Layer {
   // Provide empty definition to disable check for missing definition
   initializeState() {}
 
+  // Updates selected state members and marks the composite layer to need rerender
+  setState(updateObject) {
+    super.setState(updateObject);
+    this.setLayerNeedsUpdate();
```

Comment:
this is will make `layermanager` to re-render all existing layers right? Instead can we just iterate over the layers of the composite-layer mark them to re-draw?

Reply:
I see, I also found the note in `updateLayers` method. Thanks for the details.